src/mistral_common/protocol/instruct/normalize.py

```diff
@@ -146,9 +147,24 @@ def _aggregate_assistant_messages(self, messages: List[UATS]) -> AssistantMessag
                     tool_calls.append(normalized_tool_call)
 
             if message.content:
-                aggregated_content.append(self._aggregate_content_chunks(message.content))
-
+                if isinstance(message.content, str):
```

Comment:
Uff I don't love this. We should make use of `_aggregate_content_chunks` here? Why not use this function?